.gitignore

```diff
@@ -29,4 +29,4 @@ out/
 .DS_Store
 
 ### Playright state management
-bin/.state/**/*
\ No newline at end of file
+/bin/.state/config.json
```

Comment:
P.S. This doesn't seem to be working for me so need to take some time to figure why it is not behaving and get to fall into that space of "this file exists but ignore changes to it that are not explicitly added".